<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameshaderBrowser</class>
 <widget class="QDialog" name="FrameshaderBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Frame shader browser</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QCheckBox" name="showAllBatches">
     <property name="toolTip">
      <string>Also displays batches which are not relevant for materials.</string>
     </property>
     <property name="text">
      <string>Show all passes</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Frame shader</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="frameShaderSelect">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTreeWidget" name="frameShaderTree">
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Shader / Batch</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Output texture</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Clear flags</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="refreshButton">
     <property name="toolTip">
      <string>Reload selected frame shader from file.</string>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
